src/util/http-api.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as https from 'https';
+
+// Hmmm... no-experimental-fetch flag is used. Ok
```

Comment:
This flag is used in test because we've been using `nock` to fake http requests and it can't handle the built-in fetch :(

Reply:
Ok. Is it necessary to mock the transport layer? Maybe `http.getJson` mock would suffice here?